tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -158,13 +158,188 @@ class CComContext {
   HRESULT Init() { HRESULT hr = CoInitializeEx(0, COINIT_MULTITHREADED); if (SUCCEEDED(hr)) { m_init = true; } return hr; }
 };
 
+//= DXR Utility ============================================================================
```

Comment:
Can all this utility code be moved to somewhere after the definition of the `ExecutionTest` class?  This location is problematic for our HLK test translation path.

Reply:
No problem; moving below.